src/interface/read_config_file.py

```diff
@@ -0,0 +1,81 @@
+import xml.etree.ElementTree as etree
+class ReadConfigFile():
+    
+    def __init__(self, file_config):
+        """ The Main function(constructor) -- creates the class attributes.
+
+        file_config -- string type, the function receives the file configuration XML file name.
+        self.xmld -- import data from the XML file.
+        self.root -- reading the xml data.
+        """                         
+        self.xmld = etree.parse(file_config)             
+        self.root = self.xmld.getroot()        
+
+    def get_output_file(self):
+        """ This function returns list_output.
+
+        list list_output -- it has assigned all values read from config file. """
+        list_output = []
+        for output in self.root.findall('Output'):
+            path = output.find('Path').text
+            name = output.find('Name').text                     
+
+        list_output.append(path)
+        list_output.append(name)        
+        
+        return list_output            
+
+    def get_list_of_algorithms(self):
+        """ This functions returns list_algorithms.
+
+        list list_algorithms -- it has assigned all algorithms names read from config file.
+        """
+        list_algorithms = []
+        
+        for solve in self.root.findall('./Solve/Algorithm'):                        
+            if solve.find('Status').text == 'Active':
+                name_alg_active = solve.get('name')               
+            else:
+                name_alg = solve.get('name')
+                list_algorithms.append(name_alg)
+
+        list_algorithms.append(name_alg_active)
+        list_algorithms.reverse()
+        
+        return list_algorithms
+    
+    def get_list_of_generation_levels_names(self):
+        """ This function returns list_levels. 
+        list_levels -- list variable type, it will have assigned the levels names read from config file.
```

Comment:
What type of value is list_leve? please add examples to your parameter and return values documentation